Frontend/src/App.js

```diff
@@ -14,6 +15,9 @@ export default class App extends Component {
       <Layout>
         <Route exact path='/' component={Home} />
         <Route path='/flowers' component={AllProducts} />
+        <Route path='/blogs' component={Blogs} />
+        <Route path="/blog/:id" component={Blog} />
+        <Route path="/createBlog" component={CreateBlog} />
```

Comment:
Keep in mind, that we will have a user system with default and admin users. Where admin users will be able to see these Routes and have access to them/specific blog posts.

Reply:
Not in this story scope. Good point though.